server/routes/api.js

```diff
@@ -0,0 +1,57 @@
+
+const express = require('express')
+const router = express.Router()
+const axios = require('axios')
+const db = require('../database/dbManager.js')
+const key = "ab6e95edbd377ee1c1eef4009eea79fb"
+
+router.get('/sanity', function (request, response) {
+    console.log("Ok!")
+    response.send('Ok!')
+})
+
+router.post('/weather', async function (request, response) {
+    const weather = request.body
+    await db.saveWeatherToDB(weather)
```

Comment:
You should receive the response from the DB here (which you are returning from the function). That way you send back the actual document to the client instead of the exact same thing that they sent you.